### PR TITLE
Add more possible contributions to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Description

This PR adds a few more potential contributions that community members can make beyond those that are already mentioned within the CONTRIBUTING.md document. This was spurred on by a recent question from our Make WP Slack community.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]
Wonderful!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,3 +106,22 @@ Skip any steps you've already completed on your own. This is meant as an exhaust
 Most of these issues are potentially able to be completed in less than 4 hours, development environment set up included. It may take significantly more or less than 4 hours depending on experience and how smoothly development environment set up goes. Unfortunately sometimes dev env set up can sometimes be tricky! In these cases it would be helpful to the Openverse project to share your experience in an issue so we can try to remove any roadblocks for future contributors. **You can make meaningful contributions to the project in the form of valuable feedback about the development experience!**
 
 [List of Good First Issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+repo%3AWordPress%2Fopenverse-catalog+repo%3AWordPress%2Fopenverse+repo%3AWordPress%2Fopenverse-api+repo%3AWordPress%2Fopenverse-frontend+label%3A%22good+first+issue%22+label%3A%22help+wanted%22+-label%3A%22%E2%9B%94+status%3A+blocked%22+-label%3A%22%F0%9F%94%92+staff+only%22+)
+
+## 🏔️ Other non-code contributions
+
+There are a number of other ways to contribute to the project that don't involve writing Python or Javascript.
+
+### 📑 Text improvements
```
Comment (dhruvkb) [inline CONTRIBUTING.md:114]: Should we include info about contributing translations via GlotPress here?

**Review by dhruvkb — APPROVED** [review]
Great additions!

File: CONTRIBUTING.md (hunk 1 above)
Comment (AetherUnbound) [reply CONTRIBUTING.md:114]: Great news, that's already mentioned!
https://github.com/WordPress/openverse/blob/bf6819d857b2c5f4aef043f6668562cfe75a7b2c/CONTRIBUTING.md#-translation-contributions

**Pull request merged**